grid/chat/members/profile.supr

```diff
@@ -0,0 +1,179 @@
+"""
+Chat Members
+Retrieve members of the workspace.
+"""
+name = "chat/members"
+version = "1.0.0"
+
+"""
+Get workspace members
+Retrieve paginated list of members
+"""
+usecase GetMembers {
+  input {
+    """
+    Workspace
+    Representation of a workspace. Can be ID or name, depending on the provider capability.
+    """
+    workspace string!
+    
+    """
+    Limit
+    Maximum number of records to retrieve
+    """
+    limit number
+    
+    """
+    Page
+    Identification of page with paginated results (cursor)
+    """
+    page string!
+  }
+
+  result Members
+  error Error
+
+  example Successful {
+    input {
+      workspace = '877903817948147752',
+      limit = 2
+    }
+
+    result {
+      members = [
+        {
+          id = '1'
+          username = 'user1'
+          joinedAt = 1643798532134.069,
+        }
+        {
+          id = '2'
+          username = 'userBot'
+          joinedAt = 1643808532134.069,
+          isBot = true
+        }
+      ]
+    }
+  }
+
+  example Failed {
+    input {
+      workspace = 'not-existing',
+    }
+
+    error {
+      title = 'workspace_not_found'
+    }
+  }
+}
+
+model Member {
+  """
+  Id
+  Unique identifier of the member
+  """
+  id! string!
+
+  """
+  Username
+  Username identifier of the member
+  """
+  username! string!
+
+  """
+  Joined at
+  UNIX ms timestamp (e.g. 1095379198750) when the member joined workspace
+  """
+  joinedAt number
+
+  """
+  Is Bot
+  Boolean that represents whether the member is a bot
+  """
+  isBot boolean
+}
+
+model Members {
```

Comment:
Nitpicking: I'd inline this model into use-case, or at least rename it to `MembersResult`.